lib/syslog.ml

```diff
@@ -49,7 +49,7 @@ let facility_of_string s =
 
 exception Unknown_level of string
 let level_of_string s = 
-	match String.lowercase s with
+	match (String.lowercase s)[@ocaml.warning "-3"] with
```

Comment:
good point, I'll try to make the warning non-fatal for `jbuilder build --dev` instead, so you can still see other warnings and doesn't stop the whole build on this one.

Reply:
If you can make it, it would be great! However, afaik, the set of dev flags is hardcoded for the moment.

I'd leave that fail (we can always silence it by hand while fixing the dev build) and make the release build the default one.

Reply:
Looks like `jbuilder --dev` expands the flags into `:standard` and then I can turn some errors back into warnings with `-warn-error -N`: `(flags (:standard -w -39 %s -warn-error -3))`.
Not sure whether we want `--dev` to be the default or not though.